go/ql/integration-tests/all-platforms/go/diagnostics/build-constraints-exclude-all-go-files/diagnostics.expected

```diff
@@ -0,0 +1,15 @@
+{
+  "internal": false,
+  "markdownMessage": "Make sure the `GOOS` and `GOARCH` [environment variables are correctly set](https://docs.github.com/en/actions/learn-github-actions/variables#defining-environment-variables-for-a-single-workflow). Alternatively, [change your OS and architecture](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#using-a-github-hosted-runner).",
```

Comment:
![Uploading image.png…]()

Reply:
@simon-engledew It appears an upload of the error preview failed here ☝🏼

Reply:
Re-uploaded! Thanks! ❤️